files/workflows/ZWEWRF04.xml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<workflow xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<autoTakeOwnership>true</autoTakeOwnership>
+<!-- instance prefix settings -->
+    <workflowInfo>
+        <workflowID scope="system" isCallable="system">zowe_create_runtime</workflowID>
+        <workflowDefaultName>Zowe Create Runtime</workflowDefaultName>
+        <workflowDescription>Use this workflow to create the runtime from the convenient build.</workflowDescription>
+        <workflowVersion>1.0</workflowVersion>
+        <vendor>ZOWE</vendor>
+        <General/>
+    </workflowInfo>
+    <variable name="zowe_build" scope="instance" visibility="public">
+        <label>Zowe Build</label>
+        <abstract>Path to the extracted convenience build</abstract>
+        <description>Path to the folder from the extracted convenience build PAX file.</description>
+        <category>Build Location</category>
+        <string valueMustBeChoice="false" multiLine="false">
+            <regularExpression>~?(\/[^\/]+)+</regularExpression>
+            <errorMessage>Please specify proper full USS path ending without a slash.</errorMessage>
+            <!-- Specify choices here -->
+            <default>/u/users/ibmusers/zowe-build/zowe-1.8.0</default>
+        </string>
+    </variable>
+    <variable name="zowe_install_path" scope="instance" visibility="public">
+        <label>USS Install Path</label>
```

Comment:
add optional variable to define TMPDIR. This is used by zowe-install.sh when $INSTALL_DIR is read-only